spec/spec_helper.rb

```diff
@@ -0,0 +1,6 @@
+require 'bundler'
+
+Bundler.setup
+Bundler.require(:default, :development)
+
+require 'ph_model'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.